src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -588,7 +588,96 @@ The next four parameters are optional, and apply to advanced Azure endpoint conf
 - *staticPublicIP:* A boolean flag to determine whether the public IPs of the infrastructure's VMs should be static. The default value is _true_.
 - *additionalProperties:* Additional JVM properties to configure your ProActive node. The default values allow to handle both PNP and PAMR protocols. You can add your own properties but be aware that removing the current values might cause the deployment to fail.
 
-The Azure infrastructure offers a simple interface to easily create ProActive nodes on Azure. If your project requires a dynamic and highly customizable set of Azure VMs, you can opt for an Azure Scale Set Infrastructure.
+The Azure infrastructure offers a simple interface to easily create ProActive nodes on Azure. If your project requires a dynamic and customizable set of Azure VMs, you can opt for an <<_azure_scale_set_infrastructure>>.
+
+==== Azure Scale Set Infrastructure
+
+We provide a highly customizable infrastructure to deploy ProActive nodes using an https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/overview[Azure Scale Set]. This infrastructure will allow you to automatically adjust the number of deployed virtual machines (VM) in response to the current workload, and according to your configured parameters, e.g. min/max number of nodes, number of nodes per VM, or minimal up-time. Auto-scaling will help you increase your processing capacity during peak computation periods, while keeping your expenses low during less intensive or idle periods.
+
+Whether you require to run your workflows on Linux or on Windows, use standard or custom VM images, do some pre-configuration steps, or mount a shared file system, the Azure Scale Set infrastructure can handle it. This guide describes the requirements and the various configuration parameters available to deploy an Azure Scale Set infrastructure.
```

Comment:
I would remove the `on` for `on Windows` and the comma before `or`.
=> `Whether you require to run your workflows on Linux or Windows, use standard or custom VM images, do some pre-configuration steps or mount a shared file system, the Azure Scale Set infrastructure can handle it.`

Reply:
- Ok for the "on".
- For the comma I prefer to keep it as it really helps breaking correctly the elements on the list. I understand it's a matter of taste, it's an Oxford comma dilemma. ;)
https://english.stackexchange.com/questions/267256/oxford-comma-before-or